pgpool resolves localhost to ::1 (IPv6) and fails to bind on ::1:9999.

--- a/src/sample/pgpool.conf.sample
+++ b/src/sample/pgpool.conf.sample
@@ -24,7 +24,7 @@
 
 # - pgpool Connection Settings -
 
-listen_addresses = 'localhost'
+listen_addresses = '127.0.0.1'
                                    # Host name or IP address to listen on:
                                    # '*' for all, '' for no TCP/IP connections
                                    # (change requires restart)
--- a/src/sample/pgpool.conf.sample-logical
+++ b/src/sample/pgpool.conf.sample-logical
@@ -24,7 +24,7 @@
 
 # - pgpool Connection Settings -
 
-listen_addresses = 'localhost'
+listen_addresses = '127.0.0.1'
                                    # Host name or IP address to listen on:
                                    # '*' for all, '' for no TCP/IP connections
                                    # (change requires restart)
--- a/src/sample/pgpool.conf.sample-master-slave
+++ b/src/sample/pgpool.conf.sample-master-slave
@@ -24,7 +24,7 @@
 
 # - pgpool Connection Settings -
 
-listen_addresses = 'localhost'
+listen_addresses = '127.0.0.1'
                                    # Host name or IP address to listen on:
                                    # '*' for all, '' for no TCP/IP connections
                                    # (change requires restart)
--- a/src/sample/pgpool.conf.sample-replication
+++ b/src/sample/pgpool.conf.sample-replication
@@ -24,7 +24,7 @@
 
 # - pgpool Connection Settings -
 
-listen_addresses = 'localhost'
+listen_addresses = '127.0.0.1'
                                    # Host name or IP address to listen on:
                                    # '*' for all, '' for no TCP/IP connections
                                    # (change requires restart)
--- a/src/sample/pgpool.conf.sample-stream
+++ b/src/sample/pgpool.conf.sample-stream
@@ -24,7 +24,7 @@
 
 # - pgpool Connection Settings -
 
-listen_addresses = 'localhost'
+listen_addresses = '127.0.0.1'
                                    # Host name or IP address to listen on:
                                    # '*' for all, '' for no TCP/IP connections
                                    # (change requires restart)
